themes/bootstrap3/templates/Helpers/star-rating.phtml

```diff
@@ -0,0 +1,31 @@
+<?php
+  $id = $this->escapeHtmlAttr(uniqid());
+  $labelAttrs = !empty($this->labelAttrs) ? (' ' . implode(' ', $this->labelAttrs)) : '';
+  $inputAttrs = !empty($this->inputAttrs) ? (' ' . implode(' ', $this->inputAttrs)) : '';
+  $rating = $ratingData['count'] ? ($ratingData['rating'] ?? 0) : null;
+  // Unique name for the fields of read-only ratings so that displaying multiple ratings on a page works:
+  $fieldName = $this->readonly ? "rating_{$id}" : 'rating';
+?>
+<div class="star-rating<?=$this->readonly ? ' readonly' : ''?>">
+  <label class="rating__label hidden" for="rating_<?=$id?>_none" aria-label="<?=$this->transEsc('rating_none')?>"<?=$labelAttrs?>>
+    &nbsp;
+  </label>
+  <input class="rating__input hidden" name="<?=$fieldName?>" id="rating_<?=$id?>_none" value="" type="radio" autocomplete="off"<?=null === $rating ? 'checked' :''?><?=$inputAttrs?>>
```

Comment:
Perhaps we could turn this into a way to clear a previous rating.

Reply:
WRT the change event, I don't think that a span will fire a change event. Binding to a queried list has the problem (that we have all sorts of workarounds in other places) that it needs to be done or re-done after any possible content has loaded. In this case it can be the lightbox content or comments tab.

The rating_none field is required for accessibility. Otherwise there's no indication of the current state. If you think it's not necessary, I can check if we could do without.

It's intentionally not possible to remove a previously given rating. You can revise the rating at any time, though.

Reply:
Actually, the rating_none field is crucial so that the stars after it don't get highlighted.